share/completions/brew.fish

```diff
@@ -6,10 +11,32 @@ function __fish_brew_needs_command
   return 1
 end
 
+function __fish_brew_needs_services_action
+  __fish_brew_using_command services
+
+  set cmd (commandline -opc)
+  if test (count $cmd) -le 2
+    return 0
+  end
+  return 1
+end
+
+function __fish_brew_services
+  brew services list | awk '{if (NR>1) print $1}'
```

Comment:
If we don't find a great solution, just leave this like it is. Finding the correct directories is probably a bit involved.